README.md

```diff
@@ -257,3 +257,24 @@ Demo
 ---------------
 Demo applications are available on [pippo-demo](https://github.com/decebals/pippo-demo)   
 For a real life application built with Pippo please look at [Web Accounting - Pippo Demo](https://github.com/decebals/matilda)
+
+Contributors
```

Comment:
I cannot maintain a such list. A such list is available at https://github.com/decebals/pippo/graphs/contributors.